Source/.nuget/Pack.bat

```diff
@@ -19,7 +19,7 @@ nuget pack ..\DotSpatial.NTSExtension\DotSpatial.NTSExtension.csproj -version "%
 nuget pack ..\DotSpatial.Positioning\DotSpatial.Positioning.csproj -version "%NugetVersion%" -Properties "PackageVersion=%PackageVersion%";"Configuration=Release" 
 nuget pack ..\DotSpatial.Positioning.Design\DotSpatial.Positioning.Design.csproj -version "%NugetVersion%" -Properties "PackageVersion=%PackageVersion%";"Configuration=Release" 
 nuget pack ..\DotSpatial.Positioning.Forms\DotSpatial.Positioning.Forms.csproj -version "%NugetVersion%" -Properties "PackageVersion=%PackageVersion%";"Configuration=Release" 
-nuget pack ..\DotSpatial.Projections\DotSpatial.Projections.csproj -version "%NugetVersion%" -Properties "PackageVersion=%PackageVersion%";"Configuration=Release" 
```

Comment:
Why did you switch from csproj to nuspec?
I'd rather get rid of nuspec completely.

Reply:
@inforithmics Any comment here? This is the last thing keeping me from merging.

Reply:
The reason I changed to .nuspec was that with csrpoj and .nuspec combined the nuget package creation didn't work.
When the goal ist to get rid of .nuspec I can try if it works without .nuspec or a reduced one.

Reply:
Based on the fact that VS2022 is able to build nupkgs from csproj I'd love to get rid of the nuspecs. If you have the time I'd be grateful if you'd take a look at that :)